lib/web/apiserver_test.go

```diff
@@ -3965,6 +3967,137 @@ func TestChangeUserAuthentication_WithPrivacyPolicyEnabledError(t *testing.T) {
 	require.True(t, apiRes.PrivateKeyPolicyEnabled)
 }
 
+func TestChangeUserAuthentication_settingDefaultClusterAuthPreference(t *testing.T) {
+
+	tt := []struct {
+		cloud               bool
+		name                string
+		numberOfUsers       int
+		resultConnectorName string
+	}{{
+		cloud:               true,
+		name:                "cloud, 1 user; signing with passwordless should change connectorName to `passwordless`",
+		numberOfUsers:       1,
+		resultConnectorName: constants.PasswordlessConnector,
+	}, {
+		cloud:               false,
+		name:                "not cloud, 1 user; signing with passwordless should not change connectorName",
+		numberOfUsers:       1,
+		resultConnectorName: "",
+	}, {
+		cloud:               true,
+		name:                "cloud, 2 users; signing with passwordless should not change connectorName",
+		numberOfUsers:       2,
+		resultConnectorName: "",
+	}}
+
+	for _, tc := range tt {
+		modules.SetTestModules(t, &modules.TestModules{
+			TestFeatures: modules.Features{
+				Cloud: tc.cloud,
+			},
+		})
+
+		s := newWebSuite(t)
+
+		// auth preference
+		const RPID = "127.0.0.1"
```

Comment:
```suggestion
		const RPID = "localhost"
```

IPs are not valid RPIDs.

Reply:
0849bc1d1a0089a3ce1274bbb2c78aaff6e41953